pandas/tests/frame/methods/test_describe.py

```diff
@@ -298,3 +298,13 @@ def test_describe_percentiles_integer_idx(self):
             ],
         )
         tm.assert_frame_equal(result, expected)
+
+
+def test__unhashable_dict():
+    result = pd.DataFrame(
```

Comment:
- Could you remove the extra `_` in `test__`
- Could you add the issue number as a comment?
- This test should test this behavior: https://github.com/pandas-dev/pandas/issues/32409#issuecomment-626423254